docs(k8s): update information for iam/rbac (#5278)
scaleway/docs-content: pages/kubernetes/how-to/connect-cluster-kubectl.mdx

- `KubernetesFullAccess` (or `KubernetesReadOnly`, depending on your needs):
  Grants you the ability to manage (or list/read) Kubernetes clusters, nodes, and related actions in your Scaleway Project.

> If you do not have configured IAM for your account yet, you can generate a `kubeconfig` file with IAM and required permission sets after cluster creation from the Scaleway console.

Reviewer: ```suggestion
If you have not yet configured IAM on your account, you can generate a `kubeconfig` file with IAM and the required permission sets from the Scaleway console after creating a cluster.
```
